views/layouts/_generic.njk

```diff
@@ -46,6 +46,7 @@
 {% block footer %}{% endblock %}
 
 {% block bodyEnd %}
+<script src="{{ getFingerprint('javascripts/govuk-frontend.js') }}"></script>
```

Comment:
Why is this change required?

Reply:
This initialises the GOV.UK Frontend details element polyfill so that details works in IE/Edge. I thought I should add it as part of this PR since this is the first time we use the details element in the app itself, hence we haven't run into this before. 

However I appreciate that initialising the other govuk-frontend components here introduces an unrelated change.

I can remove the change from this PR and raise a new PR? (IE/Edge work correctly in this PR without this change).

Reply:
I think I was just surprised that we're not already including frontend's JS in the layout. If it's required and we're confident it's not already being included then happy for it to be added as part of this PR.